deploy-manage/deploy/cloud-enterprise/configure.md

```diff
@@ -9,30 +9,41 @@ mapped_pages:
 
 # Configure ECE [ece-configuring-ece]
 
-⚠️ **This page is a work in progress.** ⚠️
+Now that you have {{ece}} up and running, take a look at some of the additional features that you can configure:
 
-Now that you have Elastic Cloud Enterprise up and running, take a look at some of the additional features that you can configure:
+## Common ECE tasks
 
-* [System deployment configuration](system-deployments-configuration.md) - Best practices for ECE system deployments to ensure a highly available and resilient setup.
+* [Assign Roles to Hosts](../../../deploy-manage/deploy/cloud-enterprise/assign-roles-to-hosts.md) - Make sure new hosts can be used for their intended purpose after you install ECE on them.
+* [System deployments configuration](system-deployments-configuration.md) - Best practices for ECE system deployments to ensure a highly available and resilient setup.
 * [Configure deployment templates](configure-deployment-templates.md) - Make the most out of deployment templates by configuring ECE for your hardware and creating custom deployment templates.
```

Comment:
this one should tell us what deployment templates are

Reply:
done